### DISCUSSION TITLE
The program blocks when the interface calls other api

### DISCUSSION THREAD

**Discussion by weibangqian** [miguelgrinberg/Flask-SocketIO#1862] [Q&A]
![image](https://user-images.githubusercontent.com/42798379/183343498-de2a4d63-9720-43c4-94ee-711ac164f5f7.png)

**Comment by weibangqian** [comment]
When requesting / path, the program blocks

**Reply by miguelgrinberg** [reply]
Please don't write issues on this package when you need help solving a problem with your own code. The issues board is for bugs. Use the discussion board when you have questions. Thank you.

I don't really see what does this have to do with Socket.IO, since you are not using it for anything. Sending a request to your own server is rarely a good idea, but in any case, if your app is blocking when you do it, then it must be that the application cannot handle more than one request at once. Nothing  to do with Socket.IO, and everything to do with your choice of server and how it does concurrency.
